docs/multi-echo.rst

```diff
@@ -49,7 +53,8 @@ Collecting multi-echo EPI allows us to leverage all of the information available
 as well as additional information only available when looking at signal decay across multiple TEs.
 We can use this information to denoise the optimally combined time series.
 
-.. _Pruim et al., 2015: https://www.sciencedirect.com/science/article/pii/S1053811915001822
+.. _processing pipeline details: https://tedana.readthedocs.io/en/latest/approach.html#optimal-combination
+.. _Pruim et al. (2015): https://www.sciencedirect.com/science/article/pii/S1053811915001822
```

Comment:
Here you make it look like the line was deleted; can you just add the processing pipeline details line *below* the Prium et al reference?

Reply:
I think it's easier to interpret if the links appear in the same order as they occur in the text. In this case, it looks like the processing pipeline details link is before the Pruim reference.

Reply:
True; disregard my comment